nautobot/docs/user-guide/administration/installation/nautobot.md

```diff
@@ -15,30 +15,30 @@ These instructions will guide you through the following actions:
 !!! important
     Your database server and Redis must have been successfully installed before continuing with deployment steps. If you haven't done that yet, please visit the guide on [Installing Nautobot Dependencies](index.md#installing-nautobot-dependencies)
 
-## Choose your `NAUTOBOT_ROOT`
-
-You need to select a directory path where everything related to Nautobot will be installed. We will use this value across the documentation and it will be referred to as `NAUTOBOT_ROOT`.
+## Create the Nautobot System User
 
-We will be using this path as the home directory of the `nautobot` user.
+Create a system user account named `nautobot`. This user will own all of the Nautobot files, and the Nautobot web services will be configured to run under this account.
 
-!!! tip
-    We have selected `/opt/nautobot`, but you may use any directory you choose.
+??? abstract "Choosing Nautobot Root"
 
-Later on, we will need to set this directory path as the `NAUTOBOT_ROOT` environment variable to tell Nautobot where to find its files and settings.
+    You need to select a directory path where everything related to Nautobot will be installed. We will use this value across the documentation and it will be referred to as `NAUTOBOT_ROOT`.
 
-## Create the Nautobot System User
+    We will be using this path as the home directory of the `nautobot` user.
 
-Create a system user account named `nautobot`. This user will own all of the Nautobot files, and the Nautobot web services will be configured to run under this account.
+    !!! tip
+        We have selected `/opt/nautobot`, but you may use any directory you choose.
 
 The following command also creates the `/opt/nautobot` directory and sets it as the home directory for the user.
 
-```no-highlight
+```no-highlight title="Add system user"
 sudo useradd --system --shell /bin/bash --create-home --home-dir /opt/nautobot nautobot
 ```
 
 ## Setup the Virtual Environment
 
-A Python [virtual environment](https://docs.python.org/3/tutorial/venv.html) or *virtualenv* is like a container for a set of Python packages. A virtualenv allows you to build environments suited to specific projects without interfering with system packages or other projects.
+??? info "Python Virtual Environment"
+
+    A Python [virtual environment](https://docs.python.org/3/tutorial/venv.html) or *virtualenv* is like a container for a set of Python packages. A virtualenv allows you to build environments suited to specific projects without interfering with system packages or other projects.
 
 When installed per the documentation, Nautobot uses a virtual environment in production.
 
```

Comment:
With the above restructuring, the line below becomes the first non-hidden reference to "the `NAUTOBOT_ROOT` you chose in the previous step", which is potentially a bit confusing to the user without context?